admin_manual/configuration_files/big_file_upload_configuration.rst

```diff
@@ -95,3 +95,10 @@ Tell PHP which temp file you want it to use::
 will return memory-related errors:
 
 * ``output_buffering = 0``
+
+Configuring ownCloud
+--------------------
+
+Make sure that you don't have configured a too low ``session_lifetime`` setting within
```

Comment:
Maybe:
If you have configured the session_lifetime setting in your config/config.php file then make sure it is not too low. If unsure...
It would be useful to have a link to general doc of session_lifetime.
I guess "too low" depends on the transfer speed of clients? Maybe it is difficult to give some good guidelines here?